arches/app/media/js/utils/create-vue-application.js

```diff
@@ -7,16 +7,60 @@ import StyleClass from 'primevue/styleclass';
 import ToastService from 'primevue/toastservice';
 import Tooltip from 'primevue/tooltip';
 
-import Aura from '@primevue/themes/aura';
-
+import { definePreset } from '@primevue/themes';
 import { createApp } from 'vue';
 import { createGettext } from "vue3-gettext";
+import Aura from '@primevue/themes/aura';
 
 import arches from 'arches';
 
+export const ArchesPreset = definePreset(Aura, {
+    primitive: {
+        sky: {
```

Comment:
why the `sky` naming?

Reply:
I'm extending Aura, which has a color named Sky in the palette. I replaced the darkest value sky-950 with our existing sidebar color b/c it was very close. Didn't want to create 10 other new shades of arches blue ;)

Reply:
Ah -- I think it's worth having a short dev discussion around color naming just to be sure a good foundation is laid for this. I know that PrimeVue is set up to have many colors for many situations but I believe the Arches palette will/(should?) have like 6-8 colors 👍

Reply:
I added a few primitive tokens namespaced under arches and then linked to them from some new semantic tokens. I think this is the idea, LMK how it looks to you. I left in comments that are kind of duplicative for now but show where I extracted the values from in the legacy UI.